README.md

```diff
@@ -13,12 +13,35 @@ The MLPerf Training rules are available at [training\_rules](https://github.com/
 
 The MLPerf HPC specific rules are at [hpc\_training\_rules](https://github.com/mlcommons/training_policies/blob/master/hpc_training_rules.adoc).
 
+## Compliance
+The MLPerf logging package implements logging and compliance-checking utilities
+for MLPerf benchmarks in hpc-1.1.0 branch of the MLPerf logging repository (https://github.com/mlcommons/logging/tree/hpc-1.0-branch).
+These work for the HPC v2.0 submissions as well.
+
+To install and test compliance of your runs/submissions:
+
+```
+# Install the package into your python environment.
+# A development install (-e) is recommended for now so you can pull new updates.
+git clone -b hpc-1.0-branch https://github.com/mlperf-hpc/logging.git mlperf-logging
+pip install [--user] -e mlperf-logging
+
+# Test a full submission folder
+python3 -m mlperf_logging.package_checker <YOUR SUBMISSION_FOLDER> hpc 1.0.0
+```
+
+There is also a script that tests both compliance and summarizes the results. From the mlperf-logging directory (https://github.com/mlcommons/logging), use
+```
+./scripts/verify_for_v1.0_hpc.sh
+```
+
+ <!-- 
```

Comment:
maybe just remove all the obsolete details.

Reply:
Fixed in 7cf95be6253ad51b5d9b96b78f8757067e50331b